README.md

```diff
@@ -462,6 +462,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
 * [RESTEasy](http://resteasy.jboss.org/) - Fully certified and portable implementation of the JAX-RS specification.
 * [RestExpress](https://github.com/RestExpress/RestExpress) - Thin wrapper on the JBOSS Netty HTTP stack to provide scaling and performance.
+* [Restlet Framework](http://restlet.com/products/restlet-framework/) - The flexible and powerful pioneer REST Web API framework for Java.
```

Comment:
Pioneer because that was the first REST frameworks available 10 years or so ago.
I often use Web capitalized, but I can change this.